src/components/ui/dropdown-menu.tsx

```diff
@@ -0,0 +1,200 @@
+"use client"
+
+import * as React from "react"
```

Comment:
Starting from React 16 I think we don't have to import react in the JSX file 
```suggestion
import { ElementRef, ...etc } from "react"
```